package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "A simple restaurant website made with ES6 using Webpack, to showcase DOM manipulation by dynamically generating the entire contents of the restaurant homepage using JavaScript",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
- [ ] Kindly add the neccessary start scripts:
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```

Reply:
- Also if the optional start script is used make sure to include the `web-dev-server` package.